includes/classes/admin/mapping-wizard.php

```diff
@@ -520,6 +520,38 @@ public function map_template() {
 		$sync_items = $mapping_id && $this->_get_val( 'sync-items' );
 		$notes      = '';
 
+		$is_acf_pro_installed = class_exists('acf_pro');
+		$template_has_repeatable_fields = false;
+		if (isset($template->related->structure->groups)) {
+			foreach ($template->related->structure->groups as $group) {
+				if (isset($group->fields)) {
+					foreach ($group->fields as $field) {
+						if (isset($field->metadata->repeatable) && $field->metadata->repeatable->isRepeatable) {
+							$template_has_repeatable_fields = true;
+							break 2;
+						}
+					}
+				}
+			}
+		}
```

Comment:
Should help reduce nesting.

```suggestion
foreach($template->related->structure->groups ?? [] as $group){
    foreach ($group->fields ?? [] as $field) {
        if (isset($field->metadata->repeatable) && $field->metadata->repeatable->isRepeatable) {
            $template_has_repeatable_fields = true;
            break 2;
        }
    }
}
```